apps/air-discount-scheme/backend/src/app/modules/discount/discount.controller.ts

```diff
@@ -113,4 +115,32 @@ export class PrivateDiscountController {
       unConnectedFlights,
     )
   }
+
+  @ApiExcludeEndpoint()
```

Comment:
TODO: A malicious entity may intercept this endpoint and use it for their own gain

Reply:
Consider Role Scope here

Reply:
Correct. Added to the backlog.
Currently the resolver guards against this but a malicious entity may forge a request to this endpoint with some know-how.
For now, there is easy disaster recovery since this request logs the authenticated user into the database.